cypress/integration/journalistCanViewHeader.feature.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable no-undef */
+describe("Journalist can see a Header", () => {
+  before(() => {
+    cy.visit("/");
+  });
+  it("is expected to display Yesterday News Admin Header", () => {
```

Comment:
You're searching for a header, but in the implementation code it's a div element